src/olympia/blocklist/admin.py

```diff
@@ -317,12 +321,13 @@ def save_model(self, request, obj, form, change):
                 obj.signoff_by = request.user
             elif is_reject:
                 obj.signoff_state = BlockSubmission.SIGNOFF_REJECTED
-        else:
-            # TODO: something more fine-grained that looks at user counts
-            if waffle.switch_is_active('blocklist_admin_dualsignoff_disabled'):
-                obj.signoff_state = BlockSubmission.SIGNOFF_NOTNEEDED
 
         super().save_model(request, obj, form, change)
+
+        if obj.signoff_state == BlockSubmission.SIGNOFF_PENDING:
+            if not obj.check_needs_signoff():
```

Comment:
This would read better without the `check_` prefix.

Reply:
https://github.com/mozilla/addons-server/pull/13300/commits/d6dc7663d8f4bd45bc13d722067df570e84b0892